website/pages/docs/configuration/audit.mdx

```diff
@@ -293,4 +293,10 @@ If the request returns an error the audit log will reflect the error message.
 }
 ```
 
+### `audit` Configuration Reloads
```

Comment:
@drewbailey @krishicks this is the section I'm least sure about... are there any limitations here like sections that don't get reloaded?

Reply:
It seems like enable/disable would only be supported, but it also looks like there's a bug so it currently does not work :( Will file an issue.

Reply:
filed https://github.com/hashicorp/nomad/issues/9678

Reply:
Ok, since we're focusing on docs today I'll pull that section out and drop it into https://github.com/hashicorp/nomad/issues/9678 so that we can add it back when we fix the bug.